### PR TITLE
Helix CLI command to install web components polyfills

### PR DESCRIPTION
**Pull request by kamalgill**
This PR implements the following:
1. Leverage https://oclif.io to scaffold a robust CLI for Helix
2. Implement a `helix install-deps` CLI command
3. Allow a `-o` or `--output` flag to customize the destination folder for the `helix install-deps` cmd
4. Support a `--dry-run` flag to preview output of `install-deps` command (and to simplify unit tests)

Notes for PR reviewers -- after checking out this branch locally:
1. Run `npm install` and `npm link`.  The latter will allow you to run the `helix install-deps` command globally.  Note that consumers of the package will not need the `npm link` step.
2. Switch to a third-party repo such as the Global Dashboard and run `helix install-deps`
3. Run `helix install-deps -o public/foo` to test customizing the destination folder via CLI flags

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+# -------------------------------
+# Items below generated by oclif
+# ---
+
 # Logs
 logs
 *.log
 npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
```
Comment (CITguy) [inline .gitignore]: I think the yarn files are still valid. Please revert.

File: .gitignore [hunk 2]
```diff
@@ -51,11 +55,5 @@ typings/
 # Output of 'npm pack'
 *.tgz
 
-# Yarn Integrity file
-.yarn-integrity
```
Comment (CITguy) [inline .gitignore]: I think the yarn files are still valid. Please revert.

File: src/commands/copy-polyfills.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
+/*
+ * @fileOverview oclif-based command to copy webcomponentsjs polyfills from
+ *   node_modules to destination folder (defaults to public/assets)
+ *
+ */
+const chalk = require('chalk');
+const fs = require('fs-extra');
+const glob = require('glob');
```
Comment (CITguy) [inline src/commands/copy-polyfills.ts]: maybe `globby`?

File: src/commands/copy-polyfills.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
+/*
+ * @fileOverview oclif-based command to copy webcomponentsjs polyfills from
+ *   node_modules to destination folder (defaults to public/assets)
+ *
+ */
+const chalk = require('chalk');
```
Comment (CITguy) [inline src/commands/copy-polyfills.ts]: Convert to ESM `import`

File: test/commands/copy-polyfills.test.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
+
+describe('copy-polyfills', () => {
+  test
+    .stdout()
+    .command(['copy-polyfills'])
+    .it('runs hello', ctx => {
+      expect(ctx.stdout).to.contain('hello world')
```
Comment (CITguy) [inline test/commands/copy-polyfills.test.ts]: Update this file.

File: test/commands/hello.test.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
```
Comment (CITguy) [inline test/commands/hello.test.ts]: nix this file

File: test/mocha.opts [hunk 7]
```diff
@@ -0,0 +1,5 @@
+--require ts-node/register
+--watch-extensions ts
+--recursive
+--reporter spec
```
Comment (CITguy) [inline test/mocha.opts]: maybe tweak to `dot` (I believe) to minimize output so as not to fail on CI/CD services that fail for long logs

File: tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "importHelpers": true,
+    "module": "commonjs",
+    "outDir": "lib",
+    "rootDir": "src",
+    "strict": true,
+    "target": "es2017",
```
Comment (CITguy) [inline tsconfig.json:9]: confirm this target against Node 8

**Review by CITguy — CHANGES_REQUESTED** [review]

**Comment by CITguy** [comment]
Thoughts:

```sh
helix install [deps] [-f/--force] # copy deps if dest DNE
helix update [deps] # overwrite deps at dest
helix audit [--fix]
```

File: tsconfig.json (hunk 8 above)
Comment (kamalgill) [reply tsconfig.json:9]: https://node.green/ reports similar ES2017 support for Node 8 and 10, so I'm going to leave this line as is.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+```
+USAGE
+  $ helix help [COMMAND]
+
+ARGUMENTS
+  COMMAND  command to show help for
+
+OPTIONS
+  --all  see all commands in CLI
```
Comment (CITguy) [inline README.md:54]: Shouldn't this be default if you don't provide a `COMMAND` argument?

Comment (kamalgill) [reply README.md:54]: This readme portion was generated by oclif... I think this is the default if you skip the --all

**Comment by kamalgill** [comment]
@CITguy I think your PR comments have all been addressed.  Lemme know if I'm missing anything or if this PR is good to merge?

File: README.md (hunk 9 above)
Comment (kamalgill) [reply README.md:54]: I have confirmed this is the default if a `COMMAND` arg isn't provided... merging the PR so we don't lose track of this.

**Pull request merged**